README.md

```diff
@@ -43,19 +43,25 @@ To get started with the FMS Frontend, follow these steps:
 
     [](https://www.phind.com/search?cache=3e28c367-5d3c-4de1-90cd-97ce7d3c2738)
 
-2.  Navigate to the project directory:
+1.  Navigate to the project directory:
```

Comment:
This is a markdown convention called 'lazy numbering' and it's great. If you need to add a point in the middle of the list, you don't need to renumber all of them. It also does the different numbering for the sublist under point 3 automatically.

Learn more about it at https://google.github.io/styleguide/docguide/style.html#lists